src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -89,7 +84,7 @@ export class Message extends React.Component<Props, any> {
 
     return (
       <Container>
-        <Avatar />
+        <Avatar isUser={true} initials={this.props.initials} />
```

Comment:
Is this supposed to be ``isUser={this.props.is_from_user}` or something? Looks like it's not really being used to change the initials text color?